extensions/sql-migration/src/constants/strings.ts

```diff
@@ -82,8 +82,50 @@ export const XEVENTS_ASSESSMENT_DESCRIPTION = localize('sql.migration.database.a
 export const XEVENTS_ASSESSMENT_HELPLINK = localize('sql.migration.database.assessment.xevents.link', "Learn more");
 export const XEVENTS_ASSESSMENT_OPEN_FOLDER = localize('sql.migration.database.assessment.xevents.instructions', "Select a folder where extended events session files (.xel and .xem) are stored");
 
+// Assessment results
+export const ASSESSMENT_RESULTS_PAGE_TITLE = localize('sql.migration.assessment.results.title', "Target Platform & Assessment Results");
+export const ASSESSMENT_RESULTS_SUMMARY_LABEL_CAPS = localize('sql.migration.assessment.results.summary.label.caps', "ASSESSMENT RESULTS");
+export const ASSESSMENT_RESULTS_PAGE_HEADER = localize('sql.migration.assessment.results.header', "Choose Target platform, view assessment results and select database(s) for migration.");
+export const ASSESSMENT_RESULTS_SUMMARY_LABEL_DESCRIPTION = localize('sql.migration.assessment.results.summary.label.description', "Assessment results shows the migration readiness of the database(s)");
+export const DATABASES_ASSESSED_LABEL = localize('sql.migration.database.assessed.label', "Database(s) assessed");
+export const MIGRATION_TIME_LABEL = localize('sql.migration.migration.time.label', "Ready for migration");
+export const ASSESSMENT_FINDINGS_LABEL = localize('sql.migration.assessment.findings.label', "Assessment findings");
+export const MIGRATION_READINESS_LABEL = localize('sql.migration.migration.readiness.label', "Migration readiness");
+export const SUMMARY_TITLE = localize('sql.migration.summary.title', "Summary");
+export const DETAILS_TITLE = localize('sql.migration.details.title', "Details");
+export const ASSESSMENT_SUMMARY_TITLE = localize('sql.migration.assessment.summary.title', "Assessment summary");
+export const READINESS_SECTION_TITLE = localize('sql.migration.readiness.section.title', "Migration readiness of assessed databases in the Server instance");
```

Comment:
"Server instance" should not be capitalized.  should be sentence cased....  "...assessed databases in the server instance"